doc/api/permissions.md

```diff
@@ -563,6 +563,12 @@ Wildcards are supported too:
 After passing a wildcard character (`*`) all subsequent characters will
 be ignored. For example: `/home/*.js` will work similar to `/home/*`.
 
+When the permission model is initialized, if no wildcard is specified, it will
+be implicitly added if the current directory exists, otherwise it will not be
+implicitly added. e.g. `/home/test/files` if `files` exists as a directory then
+you can think of it as `/home//test/files/*`, otherwise the access is only
+allowed to `/home/test/files`.
```

Comment:
```suggestion

When the permission model is initialized, it will automatically add a wildcard (*) if the specified directory exists. For example, if `/home/test/files` exists, it will be treated as `/home/test/files/*`. However, if the directory does not exist, the wildcard will not be added, and access will be limited to `/home/test/files`. If you want to allow access to a folder that does not exist yet, make sure to explicitly include the wildcard: `/my-path/folder-do-not-exist/*`.
```